app/views/pages/tutorials/models/crud-models-using-ajax.liquid

```diff
@@ -277,6 +277,15 @@ updateForm.addEventListener("submit", Update);
 
 The delete operation also needs `method` and `id` because underneath it all, it is just an update.
 
+#### A note on soft and hard delete
+
+By default when you delete a record what actually happens is this: we set the value for `deleted_at` timestamp and we don't show this record in results unless you search for it explicitly.
+That is why we have `deleted_at: { exists: false }` filter available in some GraphQL queries. This is called soft delete.
```

Comment:
If default is to not show records with deleted_at, then we have `exists: true` for that reason to show them.

But i think it is not default, or at least not everywhere. There have been complaints about it.

@Slashek

Reply:
@pavelloz you are correct that's current behaviour

Reply:
So I'll update this a bit to state we show those records in some queries but not in others and that you can use `exists: true` filter when appropriate to explicitly choose which one is it but this in fact should be documented for all queries... something like "this query includes soft deleted records by default"